web/src/narrow.js

```diff
@@ -933,8 +945,19 @@ export function by_topic(target_id, opts) {
     activate(search_terms, opts);
 }
 
-// Called for the 'narrow by stream' hotkey.
```

Comment:
I couldn't find this hotkey reference, so it's probably an outdated comment, which is why I deleted it. Also, this function is called in various places at this point.

Reply:
This is probably the "Shift + S" shortcut we just got rid of in favor of "S" narrowing to the stream from topic views.